app/services/filter-service.js

```diff
@@ -52,6 +52,22 @@ export default Ember.Service.extend(Ember.Evented, {
     this.set("offerStateFilters", []);
   },
 
+  offerSorting: PERSISTENT_VAR("offerSorting", null),
+
+  init() {
+    this._super(...arguments);
+    this.set("openOfferSortOption", false);
+  },
+
+  applySortOn(column, is_desc = false) {
+    this.set("offerSorting", { column, is_desc });
+    this.toggleSortPage();
```

Comment:
if `applySortOn` is called while the "sort page" is closed, the `toggle` will open the page.
So you probably want a `set('openOfferSortOption', false)` instead